database/migrations/2025_04_08_032417_create_laravel_forms_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateLaravelFormsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('laravel_forms', function (Blueprint $table) {
+            $table->id('フォームID');
+            $table->string('名前');
+            $table->string('メールアドレス');
+            $table->integer('年齢');
+            $table->unsignedBigInteger('技術ID'); // 外部キー制約なし
+            $table->timestamps();
```

Comment:
下記のように、命名は英語でお願いします。
```suggestion
            $table->id('form_id');
            $table->string('name');
            $table->string('address');
            $table->integer('age');
            $table->unsignedBigInteger('tech_id'); // 外部キー制約なし
            $table->timestamps();
```